packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -71,6 +71,7 @@ const propTypes = {
     dataSourceId: PropTypes.string,
     dataFilter: PropTypes.objectOf(PropTypes.string),
     type: PropTypes.string,
+    isStreamingMetricEnabled: PropTypes.bool,
```

Comment:
A convention for our library is for booleans to start with `has`
```suggestion
   hassStreamingMetricEnabled: PropTypes.bool,
```